app/components/Views/confirmations/hooks/gas/useFeeCalculations.test.ts

```diff
@@ -65,6 +82,32 @@ describe('useFeeCalculations', () => {
     expect(result.current.calculateGasEstimate).toBeDefined();
   });
 
+  it('returns fee calculations but hide fiat on testnets when showFiatOnTestnets is false', async () => {
```

Comment:
```suggestion
  it('returns fee calculations but hides fiat on testnets when showFiatOnTestnets is false', () => {
```

I believe we can remove all of the `async ` from this file. Tested and it works locally

Reply:
Done - removed `async` operators too